src/ansys/fluent/core/services/field_data.py

```diff
@@ -1398,12 +1409,17 @@ def _get_pathlines_field_data(
         if kwargs.get("zones") is None:
             zones = []
         surface_ids = self.get_surface_ids(kwargs.get("surfaces"))
+        field_name = self._allowed_scalar_field_names.valid_name(
+            kwargs.get("field_name")
+        )
         fields_request = get_fields_request()
         fields_request.pathlinesFieldRequest.extend(
             self._fetched_data._pathlines_data(
-                self._allowed_scalar_field_names.valid_name(kwargs.get("field_name")),
+                field_name,
                 surface_ids,
-                additionalField=kwargs.get("additional_field_name"),
+                additionalField=kwargs.get(
+                    "additional_field_name"
+                ),  # TODO an additional field name
```

Comment:
About the TODO, we need to add the "field_name" variable?

Reply:
> About the TODO, we need to add the "field_name" variable?

Just means that I assume we can do the same treatment here. However, we aren't treating it the same way as other filed names up to now so was I was curious about why that was.

Reply:
@prmukherj Added #4017.